score-lang-compiler/src/main/java/org/openscore/lang/compiler/scorecompiler/ScoreCompilerImpl.java

```diff
@@ -50,6 +50,7 @@
 
     @Override
     public CompilationArtifact compile(Executable executable, Set<Executable> path) {
+        Validate.notEmpty(executable.getNamespace(),"Operation/Flow must have a namespace");
```

Comment:
@oritstone / @orius123  is this apporach valid? or shoyld be a varifier thing only???